news/2024/2024-03-19-bespoke-music-in-osu.md

```diff
@@ -0,0 +1,259 @@
+> 
+> So I'd say, both styles work well for me, as they have their own challenges and exploration.
+
+It may be a question affected by highly individual factors from the composition, organisation and mappooling sides, as well as different perspectives and expectations for the songs themselves. We'll leave that to future organisers to think about.
+
+[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/HowToPlayLN.png)](https://osu.ppy.sh/users/10879600)
+
+> Hi! My name is HowToPlayLN, in-game you can call me LN or index. You may know me as an ex-osu!mania mapper (especially [polyriddim](https://osu.ppy.sh/beatmapsets/1182702)), ex-mwc player 2019-2022, did some statistics and data science and made storyboards for some tournaments and now a composer. Meanwhile in real life I am doing coding and data science stuff.
+
+Like takehirotei, [HowToPlayLN](https://osu.ppy.sh/users/10879600) has also composed over 20 songs across a multitude of tournaments. Of course, the first thing to ask is *"how did it all start?"*.
+
+> If we're talking on how I started doing music, it would throwback to my first composition which I also mapped it and named it after myself ([HowToPlayLN - HowToPlayLN](https://osu.ppy.sh/beatmapsets/1602222#mania/3271953)). I think at some point some people were also joking with myself about "HowToPlayLN custom song when". Then... it's just that I decided to do one for an achievement that I would basically just be a jack-of-all-trades for a tournament scene in osu!mania or osu! in general.
+> 
+> My first tournament was actually [DDS Tournament](https://osu.ppy.sh/community/forums/topics/1418287?n=1) for which I offered a piano custom song, and then I decided to participate in [osu!taiko Malaysia Tournament](/wiki/Tournaments/OTMT) with my first tiebreaker custom song alongside with takehirotei's custom remix of my song being presented in [osu!mania LN Tournament 3](/wiki/Tournaments/o!mLN/3) ([Normal Distribution Remix](https://osu.ppy.sh/beatmapsets/1880819)), and I decided to develop the lore to go along with it and another custom song (for those who are interested, see [LNCryption](https://osu.ppy.sh/beatmapsets/1893946)) which was the reason I continued to do the custom songs in the tournament scenes.
+
+Whether by having such a rooted connection between their musical identity and osu! or otherwise, HowToPlayLN does not lack insight into how the osu! tournament music scene clicks. These are their thoughts and advice on what it's like to compose for osu! tournaments:
+
+> For me, it would be between the composer(s) and the mappers to discuss about the base information of how the song should be, like whether snap divisions on this bpm are suitable to map in this difficulty before shaping my own ideas. I would also consider the song name, the background story and sometimes I would actually encourage the mappers to try feedback or mapping the demo to determine whether it is alright for the given difficulty.
```

Comment:
```suggestion
> For me, it would be between the composer(s) and the mappers to discuss about the base information of how the song should be, like whether snap divisions on this bpm are suitable to map in this difficulty before shaping my own ideas. I would also consider the song name, the background story, and sometimes I would actually encourage the mappers to try feedback or mapping the demo to determine whether it is alright for the given difficulty.
```